specs/electra/beacon-chain.md

```diff
@@ -145,6 +151,8 @@ The following values are (non-configurable) constants used throughout the specif
 | `WITHDRAWAL_REQUEST_TYPE`    | `Bytes1('0x01')` |
 | `CONSOLIDATION_REQUEST_TYPE` | `Bytes1('0x02')` |
 
+@BERA: We introduce all of these fields, however, the Consensus Layer will only process `WITHDRAWAL_REQUEST_TYPE`. This means that if a user attempts to submit a `CONSOLIDATION_REQUEST_TYPE` or `DEPOSIT_REQUEST_TYPE`, they should expect to see no change to the system. However, their transaction will still be accepted on the execution layer.
```

Comment:
would note that `DEPOSIT_REQUEST_TYPE` EL requests should not occur on Berachain as the unforked EL clients should not find Deposit Contract events to form as CL requests. (because the event signature in Berachain's deposit contract is different)